src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,9 +35,11 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
-        cout<<"\t--robot                <robot-name>: robot name"<<endl;
+        cout<<"\t--namePrefix             <name-prefix>: prefix of the yarprobotstatepublisher ports"<<endl;
+        cout<<"\t--tfPrefix               <tf-prefix>: prefix of the published TFs"<<endl;
         cout<<"\t--model                  <file-name>: file name of the model to load at startup"<<endl;
         cout<<"\t--base-frame             <frame-name>: specify the base frame of the published tf tree"<<endl;
+        cout<<"\t--jointstates-topic      <topic-name>: source topic that streams the joint state"<<endl;
```

Comment:
The default value of `jointstates-topic`  should be documented in  this help.

Reply:
done https://github.com/robotology/idyntree/pull/538/commits/2014337cc5eebe96c22e23f33a5948e4d3c62f8d